spec/rubocop/cop/style/yoda_expression_spec.rb

```diff
@@ -36,14 +36,25 @@
     RUBY
   end
 
-  it 'registers an offense and corrects when using complex use of constants' do
+  it 'registers an offense and corrects when using suffix form of operator with constant receiver' do
     expect_offense(<<~RUBY)
-      TWO + (ONE + x)
-      ^^^^^^^^^^^^^^^ Non-literal operand (`(ONE + x)`) should be first.
```

Comment:
It seems better to keep the original test case. Is there any intention for removing it?

Reply:
My bad, never intended to remove that spec. Restored